packages/nc-gui/composables/useMultiSelect/index.ts

```
@@ -48,9 +50,15 @@ export function useMultiSelect(
   view?: MaybeRef<ViewType | undefined>,
   paginationData?: MaybeRef<PaginatedType | undefined>,
   changePage?: (page: number) => void,
+  _isGroupBy?: MaybeRef<boolean>,
+  fetchChunk?: Function,
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)